packages/xo-web/src/xo-app/dashboard/health/index.js

```diff
@@ -538,9 +566,32 @@ export default class Health extends Component {
 
   _getSrUrl = sr => `srs/${sr.id}`
 
+  _getDuplicatedMacAddresses = createCollectionWrapper(
+    createSelector(
+      () => this._getVifsByMac(),
+      vifsByMac => {
+        const duplicatedMacAddresses = []
+        for (const [macAddress, vifs] of Object.entries(vifsByMac)) {
+          if (vifs.length > 1) {
+            duplicatedMacAddresses.push(macAddress)
+          }
+        }
+        return duplicatedMacAddresses.sort()
+      }
+    )
+  )
+
   _getPoolPredicate = createSelector(
     createSelector(() => this.state.pools, resolveIds),
-    poolIds => (isEmpty(poolIds) ? undefined : item => includes(poolIds, item.$pool))
+    poolIds =>
+      isEmpty(poolIds)
+        ? undefined
+        : item => {
+            if (isArray(item)) {
+              return item.some(element => includes(poolIds, element.$pool))
+            }
+            return includes(poolIds, item.$pool)
```

Comment:
Use [conditional operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator)

IMHO, you need to create your own predicate in `_getVifsByMac` because we get all VIFs even if they are in different pools.